test/features/login.feature

```diff
@@ -0,0 +1,20 @@
+@smoke	
+Feature: Login	
+
+     Scenario: Login process	
```

Comment:
scenario names needs more information on what its actually doing (i.e. login with sysadmin i expect to log out button and take screenshot)